docs/tuning.rst

```diff
@@ -267,16 +267,52 @@ part be squashed by the TCP traffic.
 send buffer low watermark
 -------------------------
 
-libtorrent uses a low watermark for send buffers to determine when a new piece should
+libtorrent uses a watermark for send buffers to determine when a new piece should
 be requested from the disk I/O subsystem, to be appended to the send buffer. The low
-watermark is determined based on the send rate of the socket. It needs to be large
-enough to not draining the socket's send buffer before the disk operation completes.
-
-The watermark is bound to a max value, to avoid buffer sizes growing out of control.
-The default max send buffer size might not be enough to sustain very high upload rates,
-and you might have to increase it. It's specified in bytes in
+watermark needs to be large enough to not draining the socket's send buffer before the
+disk operation completes.
+``settings_pack::send_buffer_low_watermark`` defines the minimum send buffer
+target size (send buffer includes bytes pending being read from disk). For good
+and snappy seeding performance, set this fairly high, to at least fit a few
+blocks. This is essentially the initial window size which will determine how
+fast we can ramp up the send rate
+But by carefull with this setting if you have many connections because it does define
+an absolut part of the memory footprint per connection.
+
+``settings_pack::send_buffer_watermark_factor`` does raise the minimum memory footprint
```

Comment:
does raise -> raises
Also something else seems wrong with this sentence but i don't understand the context.